types/convert-string/tsconfig.json

```diff
@@ -2,8 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": ["es6"],
-        "noImplicitAny": true,
         "noImplicitThis": true,
+        "noImplicitAny": true,
```

Comment:
OK, now you're just moving around the flag within the same file (and still not swapping back the flag in the other tsconfig) - why?

Reply:
Apologies I've got really confused, didn't realised somehow this branch got mixed with `react-native-dotenv`. Hopefully this is fixed :) now.